src/ensembl/src/shared/accordion/components/AccordionItemHeading.tsx

```diff
@@ -14,8 +14,6 @@ const defaultProps = {
 
 export const SPEC_ERROR = `AccordionItemButton may contain only one child element, which must be an instance of AccordionItemButton.
 
-From the WAI-ARIA spec (https://www.w3.org/TR/wai-aria-practices-1.1/#accordion):
-
```

Comment:
Why delete this line? Without it, we are not showing where the quoted paragraph on the next line originates from.

Reply:
I guess I removed it by mistake while removing the line breaks. I've put it back now.